src/docs/quick-start/integrate-wrappers/configure-client.md

```diff
@@ -24,6 +24,23 @@ interface ClientConfig {
 }
 ```
 
+## The `ClientConfigBuilder`
+
+You can use the `ClientConfigBuilder` class present in `@polywrap/client-config-builder-js` to easily build the `ClientConfig` object:
+
+```typescript
+const config = new ClientConfigBuilder()
+  .addDefaults()
+  .build();
+
+// We must specify `noDefaults: true` because they are included with the `addDefaults()` method of the ClientConfigBuilder.
+const client = new PolywrapClient(config, { noDefaults: true });
```

Comment:
i haven't tested this, but if the user does not use `.addDefaults` and does `new PolywrapClient(config)` would have the same effect?

i think it's less messy from dev exp, eventually, they will need to add the line of `.addDefaults` (but if we go with what you wrote they'll also need to remove the `{ noDefaults: true }`, right?)

Reply:
Well the idea here is to show **how** to use the ClientConfigBuilder, and that includes the `noDefaults` option).

Yes, `new PolywrapClient(config)` would do the trick, but then the user is unaware of defaults being added.

I'll write something up quickly in this place that might satisfy both of us 😄

Reply:
Take a look now, I added a detailed explanation with another example.

Reply:
perfect, thank you!!